arm-build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Is there any way we can share this logic with `build.sh`?  There seems to be a fair amount of duplication.

Reply:
Removed, this was due to difference between my local machine and CI.